py3-yarl.yaml

```diff
@@ -1,40 +1,74 @@
-# Generated from https://pypi.org/project/yarl/
 package:
   name: py3-yarl
   version: 1.9.4
-  epoch: 0
+  epoch: 1
   description: Yet another URL library
   copyright:
     - license: Apache-2.0
   dependencies:
-    runtime:
-      - py3-multidict
-      - py3-idna
-      - py3-typing-extensions
-      - python3
+    provider-priority: 0
+
+vars:
+  pypi-package: yarl
+
+data:
+  - name: py-versions
+    items:
+      3.10: "310"
+      3.11: "311"
+      3.12: "312"
 
 environment:
   contents:
     packages:
       - build-base
       - busybox
       - ca-certificates-bundle
-      - py3-setuptools
-      - python3-dev
+      - py3-supported-cython
+      - py3-supported-expandvars
+      - py3-supported-pip
+      - py3-supported-python-dev
       - wolfi-base
 
 pipeline:
-  - uses: fetch
+  - uses: git-checkout
     with:
-      expected-sha256: 566db86717cf8080b99b58b083b773a908ae40f06681e87e589a976faf8246bf
-      uri: https://files.pythonhosted.org/packages/source/y/yarl/yarl-${{package.version}}.tar.gz
+      expected-commit: 6362ff155ba02964a5e773927412f7cf4ca23cd1
+      repository: https://github.com/aio-libs/yarl
+      tag: v${{package.version}}
 
-  - name: Python Build
-    uses: python/build-wheel
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: ${{vars.pypi-package}} installed for python${{range.key}}
+    dependencies:
+      provider-priority: ${{range.value}}
+      provides:
+        - py3-${{vars.pypi-package}}
+      runtime:
+        - py${{range.key}}-idna
+        - py${{range.key}}-multidict
+    pipeline:
+      - uses: py/pip-build-install
+        with:
+          python: python${{range.key}}
+      - uses: strip
+    test:
+      pipeline:
+        - uses: python/import
+          with:
+            python: python${{range.key}}
+            import: ${{vars.pypi-package}}
 
-  - uses: strip
+test:
+  pipeline:
+    - uses: python/import
+      with:
+        import: ${{vars.pypi-package}}
 
 update:
   enabled: true
-  release-monitor:
-    identifier: 12895
+  github:
+    identifier: aio-libs/yarl
```

Comment:
this is wrong.

Reply:
Could you clarify how it's wrong?  That does seem to be the correct repo.

Reply:
:cry: 
its not. i saw 'aio-libs' and was thinking that was the project rather than the org. like you pointed the updates at a different project.